db/migrate/20230706142000_change_claim_details_strings_to_integers2.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeClaimDetailsStringsToIntegers2 < ActiveRecord::Migration[7.0]
+  def change
+    change_column :claims, :defence_statement, 'integer USING CAST(defence_statement AS integer)'
+    change_column :claims, :prosecution_evidence, 'integer USING CAST(prosecution_evidence AS integer)'
+  end
+end
```

Comment:
if it was before the code had been merged or deployed to any of the servers it is fine to just update the migration... :)